src/pages/NewGroup.vue

```diff
@@ -0,0 +1,103 @@
+<script lang="ts" setup>
+import { storeToRefs } from 'pinia'
+import { ref } from 'vue'
+
+import Button from '/@/components/shared/Button.vue'
+import VueSelect from '/@/components/shared/VueSelect.vue'
+import type { Group } from '/@/lib/apis'
+import { useGroupStore } from '/@/stores/group'
+import { useUserStore } from '/@/stores/user'
+
+const userStore = useUserStore()
+const groupStore = useGroupStore()
+const { users } = storeToRefs(userStore)
+
+const group = ref({
+  name: '',
+  description: '',
+  budget: 0,
+  owners: [] as string[],
+  members: [] as string[]
+})
+
+async function handlePostGroup() {
+  if (
+    /^[1-9][0-9]*$|^0$/.test(group.value.budget.toString()) &&
+    group.value.name !== '' &&
+    group.value.description !== '' &&
+    group.value.owners.length > 0 &&
+    group.value.members.length > 0
+  ) {
+    const willPostGroup = {
+      name: group.value.name,
+      description: group.value.description,
+      budget: group.value.budget
+    }
+    const res: Group = await groupStore.postGroup(willPostGroup)
+    await groupStore.postGroupMember(res.id, group.value.members)
+    await groupStore.postGroupOwner(res.id, group.value.owners)
+  } else {
+    alert('全ての項目を入力してください')
+  }
+}
+</script>
+
+<template>
+  <div class="pt-8 px-60 h-full">
+    <h1 class="text-3xl text-center">グループの新規作成</h1>
+    <div class="flex flex-col justify-between mt-8 ml-12 mr-12 text-xl h-4/5">
```

Comment:
Groupsを参考になおしてみて欲しい:pray:
```suggestion
  <div class="pt-8 px-60 h-full">
    <h1 class="text-3xl text-center">グループの新規作成</h1>
    <div class="flex flex-col justify-between mt-8 ml-12 mr-12 text-xl h-4/5">
```